app/game.js

```diff
@@ -139,10 +141,15 @@ class Game {
 
     survivors.sort((a, b) => compare(a.hand, b.hand))
 
-    // @todo: handle end-in-a-draw situation
-    const winner = this.participants.find((p) => p.id === survivors[0].id)
+    let winnerCards = survivor[0].hand.cards;
+    let winnerIds = [];
+    survivors.forEach((survivor) => {
+      isTie = survivor.hand.cards.every(
+        (value, index) => value === winnerCards[index]);
```

Comment:
the card is an object in the following format: `{ suit, number }`, so only the number part should be compared